### PR TITLE
Adding my proposal link

### PR DESCRIPTION
**Pull request by tharun143**
Signed-off-by: tharun143 <tharunravuri2000@gmail.com>

### CODE REVIEW

**Comment by vchrombie** [comment]
Hi @tharun143 
Thanks for the PR.

File: GSoD-interest.md [hunk 1]
```diff
@@ -34,4 +34,4 @@ GSoD Mentors
 | Name | Email | Idea | Project Proposal |
 | --- | --- | --- | --- |
 | Your Name Here | Your Email Here |  Idea You Hoping to Work On | Link to Your Proposal |
-| Georg Link | linkgeorg@gmail.com | This is an example entry | Proposal Link |
+| Tharun Ravuri | tharunravuri2000@gmail.com | Building Documentation for D&I Badging project | [Proposal Link](https://docs.google.com/document/d/1aEFSaIYKscs0Y8t3-9Zqu3-xSGaujatNCIZH3xKa9Ho/edit?usp=sharing) |
```
Comment (vchrombie) [inline GSoD-interest.md:37]: I would suggest to revert this change and add a new column under Georg Link entry. If possible add a new empty line under your entry.

**Comment by tharun143** [comment]
@vchrombie I just added another commit to the same PR. Just making it simple

**Comment by tharun143** [comment]
@GeorgLink @germonprez @SMotaal Is that okay if I propose my proposal in mailing list for review or feedback?

**Comment by vchrombie** [comment]
Hi @tharun143, as you were around the discussion about the proposal, I would like to point this to you.

We worked on making a template for the proposal, [GSoD proposal template for CHAOSS](https://docs.google.com/document/d/1JyrkcSMmPF-dXyvqdU0fC5GkphkQAwyXBAQNSGQAUNE/edit?usp=sharing).
It would be really great if you can use it and update your existing proposal doc with the missing sections (if any).
Thanks.

**Comment by tharun143** [comment]
@vchrombie Sure. The template looks awesome and standard. 😃
